test/unit/wifi.js

```diff
@@ -197,15 +197,13 @@ module.exports['Tessel.prototype.connectToNetwork'] = {
         // Write to stdout so it completes as expected
         // Wrap in setImmediate to make sure listener is set up before emitting
         setImmediate(() => {
-          this.tessel._rps.stdout.emit('data', 'signal');
-          setImmediate(() => this.tessel._rps.emit('close'));
+          this.tessel._rps.stdout.emit('data', new Buffer('signal'));
```

Comment:
The string args will be changed to Buffers throughout, as `tessel.receive(...)` produces a Buffer.